lib/utils/config/index.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+/* Config util */
+const path = require('path');
+const os = require('os');
+const _ = require('lodash');
+const writeFileAtomic = require('write-file-atomic');
```

Comment:
Is there a way to e.g. use [`ensureFileSync`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureFile.md) from fse or any other method already available so that we don't introduce yet another dependency?